acceptancetests/repository/upgrade-series.yaml

```diff
@@ -0,0 +1,16 @@
+# upgrade-series-machines:
+services:
```

Comment:
I thought we weren't using the bundle any longer?

Reply:
Removed the bundle file!